File: Examples/VisualBasic/Files/Handling/SaveXLSFile .vb
Imports System.IO
Imports System.Web
Imports Aspose.Cells
Imports System

Namespace Files.Handling
    Public Class SaveXLSFile
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)
            Dim Respose As HttpResponse = Nothing
            ' Load your source workbook
            Dim workbook As New Workbook()
            If Respose IsNot Nothing Then
                ' Save in Excel2007 xlsx format
                workbook.Save(Respose, dataDir + "output.xls", ContentDisposition.Inline, New XlsSaveOptions())
            End If
            ' ExEnd:1
        End Sub
    End Class
End Namespace

File: Examples/CSharp/Files/Handling/SaveXLSFile.cs
using System.IO;
using System.Web;
using Aspose.Cells;
using System;

namespace Aspose.Cells.Examples.CSharp.Files.Handling
{
    public class SaveXLSFile
    {
        public static void Run()
        {
            // ExStart:1
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);
            HttpResponse Respose = null;
            // Load your source workbook
            Workbook workbook = new Workbook();
            if (Respose != null)
            {
                // Save in Excel2007 xlsx format
                workbook.Save(Respose, dataDir + "output.xls", ContentDisposition.Inline, new XlsSaveOptions());
            }
            // ExEnd:1
        }
    }
}
